liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -468,8 +469,7 @@ protected void doExecute() throws Exception {
 
         // look for a .sRunning file to check if the server has already started
         if (serverDirectory.exists()) {
-            File sRunning = new File(serverDirectory.getCanonicalPath() + "/workarea/.sRunning");
-            if (sRunning.exists()) {
+            if (ServerStatusUtil.isServerRunning(super.outputDirectory, serverName)) {
```

Comment:
Why are you referencing super?

Reply:
I see you are wanting the Liberty output directory vs the output directory you set in DevMojo. Unfortunate they are named the same. Probably should add a comment stating that.